packages/orchestrator-ui-components/src/components/WfoTable/WfoBasicTable/styles.ts

```diff
@@ -1,6 +1,7 @@
 import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
 import { css } from '@emotion/react';
 
+//TODO getWfoBasicTableStyles
```

Comment:
Could you elaborate here? Or add a ticket?

Reply:
its done already